src/ontology/uberon-edit.obo

```diff
@@ -224246,6 +224246,119 @@ intersection_of: part_of UBERON:0001987 ! placenta
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:41:02Z xsd:dateTime
+
+[Term]
+id: UBERON:8480012
+name: intersphincteric perianal space
+def: "A space located between the internal and external anal sphincter." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+synonym: "Eisenhammer’s space" EXACT [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:43:20Z xsd:dateTime
+
+[Term]
+id: UBERON:8480013
+name: ischiorectal perianal space
+def: "A large space outside the anal canal. It is delimited externally by the perianal skin and the gluteus, internally by the external sphincter and above by the levator ani muscle and internal obturator muscle. The two contralateral ischiorectal spaces communicate posteriorly around anorectal region via deep postanal space, located anteriorly to the ano-coccygeal ligament." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+synonym: "ischio-rectal perianal space" EXACT [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:46:32Z xsd:dateTime
+
+[Term]
+id: UBERON:8480015
+name: superior pelvi-rectal perianal space
+def: "A space located above the anal canal and is delimited by the superior pelvic aponeurosis below, by the peritoneum above and by the rectum internally." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:59:37Z xsd:dateTime
+
+[Term]
+id: UBERON:8480016
+name: perirenal space
+def: "A space that lies between the anterior and posterior renal fascia; it is a paired inverted cone shaped of tissue containing the kidney, the proximal portion of the collecting system, renal and perirenal vasculature, perirenal lymphatics, adrenal glands and variable amount of fat." [PMID:24065628]
+synonym: "perinephric space" EXACT [PMID:28050791]
+is_a: UBERON:0003693 ! retroperitoneal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T11:02:35Z xsd:dateTime
+
+[Term]
+id: UBERON:8480017
+name: anterior pararenal space
+def: "A space that is bounded anteriorly by the posterior parietal peritoneum, laterally by the lateroconal fascia and posteriorly by the anterior renal fascia; it contains the pancreatic gland, retroperitoneal segments of the duodenum, ascending and descending colon and the mesenteric root, as well as adipose tissue." [PMID:24065628]
```

Comment:
I wonder if we could axiomtamise the spatial part - using relations like anterior_to, posterior_to, etc.  Maybe @bvarner-ebi could look into this?